### DISCUSSION TITLE
GNPy Software to a group of participants

### DISCUSSION THREAD

**Discussion by NAIK131340** [Telecominfraproject/oopt-gnpy#469] [Q&A] [answered]
Hi @jktjkt ,

GNPy is an open source community software. We are planning to educate a group of participants in a workshop about GNPy software working and installation. Do we require to get any permission or copyrights attained from you end.

Thanking you

**Comment by jktjkt** [comment, marked as answer]
Hi @NAIK131340, thanks for the heads-up, and thanks for using GNPy. There's no need to get any permission for your workshop. In fact, I'm sure other people would appreciate a recording :).
